doc/apdu.md

```diff
@@ -209,6 +209,10 @@ accepted, the signature of the message is returned.
 
 Messages sent in more than one packet will be refused.
 
+If the `message` is a valid `baking message` (`Block` or `Consensus
```

Comment:
what happens when message is invalid baking message. Does it produce confirmation screen? The language seems to suggest that.

Reply:
```suggestion
Signing baking message(block and consensus) does not require user intevention, no confirmation screen will be displayed and the signing process is automatic.
```